src/Raven.Client/Documents/Operations/Counters/CounterBatch.cs

```diff
@@ -7,13 +7,19 @@
 
 namespace Raven.Client.Documents.Operations.Counters
 {
+    /// <summary>
+    /// Represents a batch of counter operations on multiple documents, supporting retrieval of counter values across nodes.
+    /// </summary>
     public sealed class CounterBatch
     {
         public bool ReplyWithAllNodesValues;
         public List<DocumentCountersOperation> Documents = new List<DocumentCountersOperation>();
         public bool FromEtl;
     }
 
+    /// <summary>
+    /// Represents counter operations for a specific document, such as increment or set operations on named counters.
+    /// </summary>
     public sealed class DocumentCountersOperation
     {
         public List<CounterOperation> Operations;
```

Comment:
What about other `public`s here?
does `DocumentId` is mandatory?

Reply:
done